templates/taxi/manufacturer_list.html

```diff
@@ -1,33 +1,41 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Manufacturer List
-    </h1>
+  <h1>Manufacturer List
+    <a href="{% url 'taxi:manufacturer-create' %}">+</a>
+  </h1>
+  {% if manufacturer_list %}
+    <table class="table">
+      <tr>
+        <th>ID</th>
+        <th>Name</th>
+        <th>Country</th>
+        <th>Update</th>
+        <th>Delete</th>
+      </tr>
 
-    {% if manufacturer_list %}
-      <table class="table">
+      {% for manufacturer in manufacturer_list %}
         <tr>
-          <th>ID</th>
-          <th>Name</th>
-          <th>Country</th>
-        </tr>
+          <td>
+            {{ manufacturer.id }}
+          </td>
+          <td>
+            {{ manufacturer.name }}
+          </td>
+          <td>
+            {{ manufacturer.country }}
+          </td>
+          <td>
+            <a href="{% url 'taxi:manufacturer-update' pk=manufacturer.id %}">Update</a>
+          </td>
+          <td>
+            <a href="{% url 'taxi:manufacturer-delete' pk=manufacturer.id %}">Delete</a>
```

Comment:
Recommendation: It would be a good idea to add a confirmation prompt when a user tries to delete a manufacturer. This can prevent accidental deletions.